CadRevealRvmProvider.Tests/Converters/RvmBoxConverterTests.cs

```diff
@@ -9,7 +9,7 @@
 [TestFixture]
 public class RvmBoxConverterTests
 {
-    const int _treeIndex = 1337;
+    const int TreeIndex = 1337;
```

Comment:
Maybe set it to private explicitly?

Reply:
If you decide it is worth it, there are a few other similar cases in the converter tests as well